samples/e2eTestUtils/jest-puppeteer-utils/puppeteer_environment.js

```diff
@@ -1,19 +1,19 @@
 const puppeteer = require('puppeteer');
-const NodeEnvironment = require('jest-environment-node');
+const NodeEnvironment = require('jest-environment-node').TestEnvironment;
 
 class PuppeteerEnvironment extends NodeEnvironment {
-    constructor(config) {
-        super(config);
-    }
+	constructor({ globalConfig, projectConfig }, context) {
```

Comment:
Custom environment change: https://jestjs.io/docs/28.x/upgrading-to-jest28#custom-environment